src/axom/slam/tests/slam_set_RangeSet.cpp

```diff
@@ -32,10 +32,10 @@ using SetType = axom::slam::RangeSet<>;
 using SetPosition = SetType::PositionType;
 using SetElement = SetType::ElementType;
 
-static const SetPosition MAX_SIZE = 20;
-static const SetElement lowerIndex = static_cast<SetElement>(.3 * MAX_SIZE);
-static const SetElement upperIndex = static_cast<SetElement>(.7 * MAX_SIZE);
-static const SetElement range = upperIndex - lowerIndex;
+constexpr SetPosition MAX_SIZE = 20;
+constexpr SetElement lowerIndex = static_cast<SetElement>(.3 * MAX_SIZE);
```

Comment:
```suggestion
constexpr SetElement lowerIndex(.3 * MAX_SIZE);
```

Does this do the same thing? If so, it is less verbose.

Reply:
I think it probably makes sense to keep the `static_cast` here do explicitly highlight the cast from double